modules/aws-eks-self-managed-node-groups/data.tf

```diff
@@ -42,3 +48,164 @@ data "aws_iam_policy_document" "eks_windows_cni" {
     ]
   }
 }
+
+data "aws_iam_policy_document" "worker_key" {
```

Comment:
I think we should let customers to pass the KMS key ARN if not we will use managed KMS key